app/editor/src/store/hooks/admin/useTopics.ts

```diff
@@ -45,12 +52,14 @@ export const useTopics = (): [IAdminState, ITopicController] => {
       },
       addTopic: async (model: ITopicModel) => {
         const response = await dispatch<ITopicModel>('add-topic', () => api.addTopic(model));
+        let items: ITopicModel[] = [];
+
         store.storeTopics((topics) => {
-          var items = [...topics];
+          items = [...topics];
           items.splice(model.sortOrder, 0, response.data);
           return items;
         });
-        await lookup.getLookups();
```

Comment:
this line is out of place here.  we should not have to retrieve ALL the lookups just because we added one new topic